nyxx/lib/src/utils/EntityUtility.dart

```diff
@@ -0,0 +1,139 @@
+part of nyxx;
+
+/// Entity Utility is a Utility that lets you create Entities from outside of the nyxx project. Typically used in nyxx.* projects.
+/// An example getting a user is below:
+/// ```dart
+/// void main() {
+///     var bot = Nyxx("TOKEN");
+///     Map<String, dynamic> json = /* JSON from the API */
+///     User user = EntityUtility.createUser(bot, json);
+/// }
+/// ```
+class EntityUtility {
+  /// Creates a User object, can be used for other classes where you have correct JSON data from the API.
+  /// ```dart
+  /// void main() {
+  ///     var bot = Nyxx("TOKEN");
+  ///     Map<String, dynamic> json = /* JSON from the API */
+  ///     User user = EntityUtility.createUser(bot, json);
+  /// }
+  /// ```
+  User createUser(
+    Nyxx client,
+    Map<String, dynamic> json,
```

Comment:
```suggestion
    Map<String, dynamic> rawJson,
```
Same goes for the rest

Reply:
Changed in next push :)